tools/github-bot/src/tools.ts

```diff
@@ -1,5 +1,8 @@
 import { Probot } from "probot";
 
+export const isValidBotPR = (branch: string): boolean =>
+  /^(dependabot)\/.+/.test(branch);
```

Comment:
`(dependabot)bad-actor` entered the room

Reply:
🥷